locales/PL_pl.json

```
@@ -0,0 +1,238 @@
+			}
+
+		]
+	},
+	"credits": {
+		"title": "Nasza drużyna",
+		"text": "Poznaj drużynę, która tworzy projekt",
+		"people": [
+			{
+				"name": "Jonathan Barrow (jonbarrow)",
+				"caption": "Właściciel projektu i główny programista",
+				"picture": "https://github.com/jonbarrow.png",
+				"github": "https://github.com/jonbarrow"
+			},
+			{
+				"name": "Jemma (CaramelKat)",
+				"caption": "Research w sprawie Miiverse oraz programowanie",
+				"picture": "https://github.com/caramelkat.png",
+				"github": "https://github.com/CaramelKat"
+			},
+			{
+				"name": "Rambo6Glaz",
+				"caption": "Instalator sieci i research w sprawie konsoli",
+				"picture": "https://github.com/Rambo6Glaz.png",
+				"github": "https://github.com/Rambo6Glaz"
+			},
+			{
+				"name": "quarky",
+				"caption": "Research w sprawie BOSS oraz programowanie łatek",
+				"picture": "https://github.com/QuarkTheAwesome.png",
+				"github": "https://github.com/QuarkTheAwesome"
+			},
+			{
+				"name": "SuperMarioDaBom",
+				"caption": "Research w sprawie konsoli i innych systemów",
+				"picture": "https://github.com/supermariodabom.png",
+				"github": "https://github.com/SuperMarioDaBom"
+			},
+			{
+				"name": "Jip Fr",
+				"caption": "Kierownik programowania webowego",
+				"picture": "https://github.com/jipfr.png",
+				"github": "https://github.com/jipfr"
+			},
+			{
+				"name": "monty",
+				"caption": "Programowanie webowe",
+				"picture": "https://github.com/ashmonty.png",
+				"github": "https://github.com/ashmonty"
+			},
+			{
+				"name": "mrjvs",
+				"caption": "Projektant",
+				"picture": "https://github.com/mrjvs.png",
+				"github": "https://github.com/mrjvs"
+			}
+		]
+	},
+	"specialThanks": {
+		"title": "Specjalne podziękowania",
+		"text": "Bez nich, Pretendo nie byłoby w takim stanie jak teraz.",
+		"people": [
+			{
+				"name": "superwhiskers",
+				"caption": "Zaprogramowanie biblioteki crunch",
+				"picture": "https://github.com/superwhiskers.png",
+				"github": "https://github.com/superwhiskers"
+			},
+			{
+				"name": "Stary",
+				"caption": "Programista 3DS oraz dysektor NEX",
+				"picture": "https://github.com/Stary2001.png",
+				"github": "https://github.com/Stary2001"
+			},
+			{
+				"name": "Billy",
+				"caption": "Archiwista danych",
+				"picture": "https://github.com/InternalLoss.png",
+				"github": "https://github.com/InternalLoss"
+			},
+			{
+				"name": "Shutterbug2000",
+				"caption": "Research w sprawie Mario Kart 7 i 3DS",
+				"picture": "https://cdn.discordapp.com/avatars/191370953807233024/0311b61e2009c1576828dd2e9a59d72e.png?size=128",
+				"github": "https://github.com/shutterbug2000"
+			},
+			{
+				"name": "Kinnay",
+				"special": "Specjalne podziękowania",
+				"caption": "Research w sprawie struktur danych Nintendo",
+				"picture": "https://cdn.discordapp.com/avatars/186572995848830987/b55c0d4e7bfd792edf0689f83a25d8ea.png?size=128",
+				"github": "https://github.com/Kinnay"
+			},
```

NinStar's strings are missing in your locale. Could you add them?
```suggestion
			{
				"name": "NinStar",
				"caption": "Icons for the Mii Editor and Juxt reactions",
				"picture": "https://github.com/ninstar.png",
				"github": "https://github.com/ninstar"
			},
			{
```

---

I suck at GitHub PR management, but the translation should be:
"Ikony do edytora Mii oraz reakcji do Juxt"